React/Naming Cheatsheet.md

```diff
@@ -0,0 +1,22 @@
+## Naming Cheatsheet
```

Comment:
We don't need a root heading, handbook will use name of the file as the first heading. We can remove this and change all other levels to `level - 1`. E.g., `### Naming Components -> ## Naming Components`.